coalib/output/printers/ListLogPrinter.py

```diff
@@ -19,14 +19,8 @@ def __init__(self,
 
         self.logs = []
 
-    def log_message(self, log_message, **kwargs):
-        if not isinstance(log_message, LogMessage):
-            raise TypeError("log_message should be of type LogMessage.")
-
-        if log_message.log_level < self.log_level:
-            return
-
-        self.logs.append(log_message)
+    def _print_log_message(self, log_message, **kwargs):
```

Comment:
`_print_log_message` have just 2 args, that's why the tests failed! `self` isn't seen as an argument when you use that function. That's the only problem rised by the tests! I hope this will help you!